core/pom.xml

```diff
@@ -719,7 +719,8 @@ THE SOFTWARE.
         <artifactId>maven-surefire-plugin</artifactId>
         <!-- Version specified in grandparent POM -->
         <configuration>
-          <argLine>@{jacocoSurefireArgs}</argLine>
+          <!-- Make sure to keep the directives in test/pom.xml and war/pom.xml in sync with these. -->
```

Comment:
Just seems to be shuffled around from a profile, but while we are here and since it is related to Java 11: would it be possible to introduce a POM property in root `pom.xml` to enforce that?

Reply:
The versions in `core/pom.xml` and `test/pom.xml` could be deduplicated but the version in `war/pom.xml` cannot be deduplicated because it is in the form of a `MANIFEST.MF` directive rather than command-line arguments. I am not sure the savings of 3 to 2 copies of the code is worth it.